### PR TITLE
feat(jest): 14% coverage for store/files

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

- 43% coverage for store/files/mutations
- 100% coverage for store/files/state
- Progress on coverage for store/files/getters

**Which issue(s) this PR fixes** 🔨

No ticket

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by stavares843 — APPROVED** [review]

**Review by WanderingHogan — APPROVED** [review]

File: store/files/state.test.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import InitialFilesState from './state'
+
+describe('Test files/state', () => {
+  it('should match the initial state snapshot', () => {
+    expect(InitialFilesState).toMatchSnapshot()
```
Comment (WanderingHogan) [inline store/files/state.test.ts:5]: this is kind of weird - just checking that it's a function when the next text accomplishes this, can you remove this one

**Review by WanderingHogan — CHANGES_REQUESTED** [review]

File: store/files/state.test.ts (hunk 1 above)
Comment (drepram) [reply store/files/state.test.ts]: Removed for duplicate!

**Comment by stavares843** [comment]
Screenshot 2022-09-07 at 13 36 36

remove the snapshot pls  🔨

**Comment by drepram** [comment]
> Screenshot 2022-09-07 at 13 36 36
> 
> remove the snapshot pls 🔨

Please check again 😄

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**